### ISSUE TITLE
#sidebar #admin-menu - links and icons

### ISSUE THREAD

**Issue by uda** [excedalogic/metro_redmine#3]
the admin menu looks great with the icons, but when displayed as sidebar, the sidebar rule of padding-left: 0px !important; takes over and the text is over the icons.
![2013-09-16 16 05 18 001](https://f.cloud.github.com/assets/199314/1148916/c6695380-1ed0-11e3-97c3-6a65d3423b60.png)

**Comment by carstencodes** [comment]
I agree.  I made a local fix overwriting the sidebar padding using a padding left of 32px. I noticed this bug on FF24 and Iron/Chrome

**Comment by astout** [comment]
fixed in https://github.com/astout/metro_redmine
